src/hooks/useDocsURL.ts

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+
+import { frameworkMap } from '../common/constants';
+import { Strings } from '../resources/Strings';
+import { Framework } from '../types/types';
+
+type SupportedPerfdocsFramework = 'talos' | 'awsy' | 'devtools';
+const supportedPerfdocsFrameworks: Record<SupportedPerfdocsFramework, string> =
+  {
+    talos: 'talos',
+    awsy: 'awsy',
+    devtools: 'performance-tests-overview',
+  };
+
+const useDocsURL = (suite: string, framework_id: Framework['id']) => {
+  const [docsURL, setdocsURL] = useState('');
+  const [isLinkSupported, setIsLinkSupported] = useState(true);
+
+  useEffect(() => {
```

Comment:
`useEffect` lets you perform side effects in function components, I don't think it makes sense to use it outside of a component.